tests/conftest.py

```diff
@@ -32,11 +32,37 @@ def pytest_configure(config):
         "skip_offline: mark the test to be skipped when the tests are run offline "
         "(this test explicitly / semantically needs an internet connection)",
     )
+    config.addinivalue_line(
+        "markers",
+        "only_in_docker: mark the test as running only in Docker (e.g., requires installation of system packages)",
+    )
+    config.addinivalue_line(
+        "markers",
+        "resource_heavy: mark the test as resource-heavy, e.g., downloading very large external dependencies, "
+        "or requiring high amount of RAM/CPU (can be systematically sampled/optimized in the future)",
+    )
     config.addinivalue_line(
         "markers",
         "aws_validated: mark the test as validated / verified against real AWS",
     )
 
+    config.addinivalue_line(
+        "markers",
+        "only_localstack: mark the test as inherently incompatible with AWS, e.g. when testing localstack-specific features",
+    )
+    config.addinivalue_line(
+        "markers",
+        "should_be_aws_validated: test fails against AWS but it shouldn't. Might need refactoring, additional permissions, etc.",
```

Comment:
What about `todo_aws_validated`, `fix_aws_validated`, or `needs_to_be_aws_validated` ?

"Should" could be misunderstood as "it should work" but not being sure 🤷‍♂️

Reply:
good idea 👍